src/Compilers/CSharp/Portable/Syntax/SyntaxFactory.cs

```diff
@@ -664,6 +665,532 @@ public static SyntaxToken XmlEntity(SyntaxTriviaList leading, string text, strin
         }
 
         /// <summary>
+        /// Creates an xml documentation comment that abstracts xml syntax creation.
+        /// </summary>
+        /// <param name="content">
+        /// A list of xml node syntax that will be the content within the xml documentation comment
+        /// (e.g. a summary element, a returns element, exception element and so on).
+        /// </param>
+        public static DocumentationCommentTriviaSyntax DocumentationComment(params XmlNodeSyntax[] content)
+        {
+            return DocumentationCommentTrivia(SyntaxKind.SingleLineDocumentationCommentTrivia, List(content))
+                .WithLeadingTrivia(DocumentationCommentExterior("/// "))
+                .WithTrailingTrivia(EndOfLine(Environment.NewLine));
+        }
+
+        /// <summary>
+        /// Creates a summary element within an xml documentation comment.
+        /// </summary>
+        /// <param name="content">A list of xml node syntax that will be the content within the summary element.</param>
+        public static XmlElementSyntax XmlSummaryElement(params XmlNodeSyntax[] content)
+        {
+            return XmlSummaryElement(List(content));
+        }
+
+        /// <summary>
+        /// Creates a summary element within an xml documentation comment.
+        /// </summary>
+        /// <param name="content">A list of xml node syntax that will be the content within the summary element.</param>
+        public static XmlElementSyntax XmlSummaryElement(SyntaxList<XmlNodeSyntax> content)
+        {
+            return XmlMultiLineElement("summary", content);
```

Comment:
All of the string literals used here should use the constants from [DocumentationCommentXmlNames](http://source.roslyn.io/#Microsoft.CodeAnalysis/InternalUtilities/DocumentationCommentXmlNames.cs).